src/callbacks_step/time_series_dg.jl

```diff
@@ -5,8 +5,10 @@
 @muladd begin
 #! format: noindent
 
-# Store time series file for a TreeMesh with a DG solver
-function save_time_series_file(time_series_callback, mesh::TreeMesh, equations, dg::DG)
+# Store time series file for a DG solver
+function save_time_series_file(time_series_callback,
+                               mesh::Union{TreeMesh, UnstructuredMesh2D},
```

Comment:
What would be missing to support also P4estMesh (in 2D and 3D)? It would be great if you could create a short issue outlining the steps, such that we can maybe have a student work on this...

Reply:
I opened the issue #1869 to document this.

Reply:
This is great, thanks a lot!